### PR TITLE
Add new translator for pressian.com - a KR news site

### PR DESCRIPTION
**Pull request by mete0r**
Add new translator for http://pressian.com, which is a online news site in South Korea.

This uses the translator framework.

### CODE REVIEW

File: pressian.com.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+	"translatorID": "3f7b7b18-1152-40b6-b9e4-e397f913641b",
+	"label": "pressian.com",
+	"creator": "mete0r@sarangbang.or.kr",
+	"target": "^https?://www.pressian.com/",
```
Comment (aurimasv) [inline pressian.com.js]: Escape periods. Also, based on your detection code below (`detect`), you should just include that part in the target regexp. In Scaffold, it should be `^https:?//www\.pressian\.com/news/article\.html`

The `detect` part can then just be `function() { return true; }` (I think)

File: pressian.com.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+	"minVersion": "1.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "g",
```
Comment (aurimasv) [inline pressian.com.js]: As @zuphilip says, you can tick all the boxes here.

File: pressian.com.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+function detectWeb(doc, url) { return FW.detectWeb(doc, url); }
+function doWeb(doc, url) { return FW.doWeb(doc, url); }
+
+
+FW.Scraper({
+itemType	: 'newspaperArticle',
+detect		: FW.Url().match(/\/news\/article\.html/),
+title		: FW.Xpath("//title/text()").text(),
+date		: FW.Xpath('//p[@class="arvdate"]/text()').text(),
+creators	: FW.Xpath('//p[@class="arvdate"]/a/span').text().replace(/ .*/,'').cleanAuthor('author'),
```
Comment (aurimasv) [inline pressian.com.js]: Are you sure this is right? You're effectively only taking the first word in the span. Also, if the span starts with a space (not uncommon to have extra whitespace in HTML) you'd be clearing everything.

File: pressian.com.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+function detectWeb(doc, url) { return FW.detectWeb(doc, url); }
+function doWeb(doc, url) { return FW.doWeb(doc, url); }
+
+
+FW.Scraper({
+itemType	: 'newspaperArticle',
+detect		: FW.Url().match(/\/news\/article\.html/),
+title		: FW.Xpath("//title/text()").text(),
+date		: FW.Xpath('//p[@class="arvdate"]/text()').text(),
```
Comment (aurimasv) [inline pressian.com.js]: nit: don't need `text()` inside the xpath

File: pressian.com.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+creators	: FW.Xpath('//p[@class="arvdate"]/a/span').text().replace(/ .*/,'').cleanAuthor('author'),
+publicationTitle: "프레시안",
+attachments	: [{
+	url: FW.Url().replace(/article/, 'article_print'),
+	title: 'Printable',
+	type: 'text/html'
+}]
+});
```
Comment (aurimasv) [inline pressian.com.js]: Please add a test case (see last tab in Scaffold)

File: pressian.com.js [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+FW.Scraper({
+itemType	: 'newspaperArticle',
+detect		: FW.Url().match(/\/news\/article\.html/),
+title		: FW.Xpath("//title/text()").text(),
+date		: FW.Xpath('//p[@class="arvdate"]/text()').text(),
+creators	: FW.Xpath('//p[@class="arvdate"]/a/span').text().replace(/ .*/,'').cleanAuthor('author'),
+publicationTitle: "프레시안",
+attachments	: [{
+	url: FW.Url().replace(/article/, 'article_print'),
+	title: 'Printable',
```
Comment (aurimasv) [inline pressian.com.js]: Call this "Snapshot"

**Comment by AbeJellinek** [comment]
Closing this for now because it uses FW, unfortunately.

**Pull request closed without merging**